Ginger/Ginger/Actions/ActionEditPage.xaml.cs

```diff
@@ -800,6 +800,17 @@ public bool ShowAsWindow(eWindowShowStyle windowStyle = eWindowShowStyle.Free, b
                     winButtons.Add(saveBtnAnalyzer);
                     winButtons.Add(closeBtnAnalyzer);
                     break;
+                case General.RepositoryItemPageViewMode.Standalone:
+                    title = "Edit " + RemoveActionWord(mAction.ActionDescription) + " Action";
+                    Button saveButton = new Button();
+                    saveButton.Content = "Save";
+                    saveButton.Click += new RoutedEventHandler(saveButton_Clicked);
+                    Button uncoButton = new Button();
+                    uncoButton.Content = "Undo & Close";
+                    uncoButton.Click += new RoutedEventHandler(undoBtn_Click);
+                    winButtons.Add(saveButton);
+                    winButtons.Add(uncoButton);
+                    break;
                 case General.RepositoryItemPageViewMode.View:
```

Comment:
why "Standalone" was needed to be added? for which scenario it is needed? it supposed to be Standalone only when exsit in Shared Repository and you already have sace for it.

Reply:
The current case for .child show a "save" button as we need,
But show a message regarding the analyzer, "Saving auto fixed issues..." which is not correct in our case.

So yes we need the same functionality as for the .child case - only save button
but the message when asking if to save is not correct, so I have added another case for it - standalone. maybe the name "standalone" is not correct, or I can check something else before showing the message if we need to show message for the analyzer or just regular save message:

        private void saveBtnAnalyzer_Click(object sender, RoutedEventArgs e)
        {
            if (Reporter.ToUser(eUserMsgKeys.SaveAnalyzerItemWarning) == MessageBoxResult.Yes)
            {
                Reporter.ToGingerHelper(eGingerHelperMsgKey.AnalyzerSavingFixedIssues, null, mAction.Description);
                saveWasDone = true;
                mActParentBusinessFlow.Save();
                Reporter.CloseGingerHelper();
            }

            IsPageClosing = true;
            _pageGenericWin.Close();
        }